Linux 6.12.101 backported "wifi: cfg80211: pass net_device to
.set_monitor_channel" from 6.13, buildroot's 0001-Fix-build-with-Linux-6.13.patch
only checks for 6.13.

--- a/ioctl_cfg80211.c
+++ b/ioctl_cfg80211.c
@@ -4404,7 +4404,7 @@
 }
 
 static int cfg80211_rtw_set_monitor_channel(struct wiphy *wiphy
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6, 13, 0))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6, 12, 101))
 	, struct net_device *dev
 	, struct cfg80211_chan_def *chandef
 #elif (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 8, 0))
